src/components/stepOne/index.js

```diff
@@ -56,16 +90,37 @@ export class stepOne extends React.Component {
   componentDidMount() {
     checkWeb3(this.props.web3Store.web3);
 
+    let downloadContracts = null
+
     switch (this.props.contractStore.contractType) {
       case CONTRACT_TYPES.standard:
-        this.getStandardCrowdsaleAssets();
+        downloadContracts = this.getStandardCrowdsaleAssets();
         break;
       case CONTRACT_TYPES.whitelistwithcap:
-        this.getWhiteListWithCapCrowdsaleAssets();
-        break;
+        downloadContracts = this.getWhiteListWithCapCrowdsaleAssets();
+        break
       default:
-        break;
+        break
     }
+
+    downloadContracts
+      .then(
+        () => {
+          this.setState({
+            contractsDownloaded: DOWNLOAD_STATUS.SUCCESS
+          })
+        },
+        (e) => {
+          console.error('Error downloading contracts', e)
+          toast.showToaster({
+            type: TOAST.TYPE.ERROR,
+            message: 'The contracts could not be downloaded'
```

Comment:
Maybe, to add "Please try to refresh the page. If the problem still persists, try again later".

Reply:
Updated.